.travis.yml

```diff
@@ -0,0 +1,14 @@
+language: ruby
+sudo: false
+rvm:
+- 2.3.1
```

Comment:
Want to do actually check against all the latest supported Rubies? 2.2.7, 2.3.4, 2.4.2 ?